macOS/DuckDuckGo/Updates/UpdateController.swift

```diff
@@ -234,33 +238,50 @@ final class UpdateController: NSObject, UpdateControllerProtocol {
     // This is the default behavior
     func checkForUpdateRespectingRollout() {
         Task { @UpdateCheckActor in
-            await performUpdateCheck()
+            await performUpdateCheck(skipRollout: false)
         }
     }
 
     @UpdateCheckActor
-    private func performUpdateCheck() async {
-        // Check if we can start a new check (Sparkle availability + rate limiting)
-        guard await updateCheckState.canStartNewCheck(updater: updater) else {
-            Logger.updates.debug("Update check skipped - not allowed by Sparkle or rate limited")
+    private func performUpdateCheck(skipRollout: Bool = false) async {
```

Comment:
Unified both `performUpdateCheck` methods into 1, since most logic was the same.